demisto_sdk/commands/common/content_constant_paths.py

```diff
@@ -20,21 +20,26 @@
 )
 NATIVE_IMAGE_PATH = CONTENT_PATH / "Tests" / NATIVE_IMAGE_FILE_NAME
 
+COMMON_SERVER_PYTHON_PATH = (
+    CONTENT_PATH / "Packs" / "Base" / "Scripts" / "CommonServerPython"
+)
+DEMISTO_MOCK_PATH = CONTENT_PATH / TESTS_DIR / "demistomock"
+API_MODULES_SCRIPTS_DIR = CONTENT_PATH / "Packs" / "ApiModules" / "Scripts"
 
 PYTHONPATH = [
-    Path(CONTENT_PATH).absolute(),
-    Path(CONTENT_PATH / "Packs" / "Base" / "Scripts" / "CommonServerPython").absolute(),
-    Path(CONTENT_PATH / TESTS_DIR / "demistomock").absolute(),
-    Path(__file__).parent.parent / "lint" / "resources" / "pylint_plugins",
+    path.absolute()
+    for path in [
+        Path(CONTENT_PATH),
+        COMMON_SERVER_PYTHON_PATH,
+        DEMISTO_MOCK_PATH,
+        Path(__file__).parent.parent / "lint" / "resources" / "pylint_plugins",
+    ]
 ]
-try:
-    PYTHONPATH.extend(
-        (
-            path.absolute()
-            for path in Path(
-                CONTENT_PATH / "Packs" / "ApiModules" / "Scripts"
-            ).iterdir()
-        )
+
+if API_MODULES_SCRIPTS_DIR.exists():
+    PYTHONPATH.extend(path.absolute() for path in API_MODULES_SCRIPTS_DIR.iterdir())
+
+else:
+    logger.debug(
+        "Could not add API modules to 'PYTHONPATH' as the base directory does not exist."
```

Comment:
This is set to `debug` level and not `warning` since this module is loaded for all commands and not just the `setup-env` command (this log is added at the start of any `demisto-sdk` command).

Reply:
does it mean that api modules will not be auto detected?

Reply:
Only if `API_MODULES_SCRIPTS_DIR.exists()` equals `False` for some reason, like when using a custom content repo (this is just to gracefully handle errors).